architecture/index.adoc

```diff
@@ -0,0 +1,11 @@
+[id="acs-architecture"]
+= {product-title} architecture
+include::modules/common-attributes.adoc[]
+:context: acs-architecture
+
+toc::[]
+
+[.lead]
```

Comment:
Minor, but I don't think we use this `[.lead]` concept. Usually we'd want an assembly's intro here to just be a regular sentence. 

But I don't know if this is something we need to worry about right now.